irg_gazebo_plugins/materials/scripts/irg_gazebo_plugins.material

```diff
@@ -24,8 +24,8 @@ fragment_program IRGCameraSimFS glsl
     param_named exposure float 1.0
     param_named gamma float 1.0
     param_named energy_conversion float 1.0
-    param_named read_noise float 0.64
-    param_named shot_noise float 0.09
+    param_named read_noise_std_dev float 0.8
+    param_named shot_noise_coeff float 0.3
```

Comment:
Is **IRGCameraNoise** used along with **IRGCameraSim**? Do we need to apply the same renaming to the `read_noise` and `shot_noise` variable defined in **IRGCameraNoise** or are these different coefficients?

Reply:
The IRGCameraNoisePlugin has been replaced by IRGCameraSimPlugins. I'd like to just remove it completely in another ticket if nobody has any complaints.

Reply:
Isn't this going to break the current settings we have for current OceanWATERS worlds? for example: https://github.com/nasa/ow_europa/blob/6e3006023b7112c6e342c68da39699bbb7a28165/worlds/atacama_y1a.world#L54-L55

Reply:
If so, then I think you should have a related PR on **ow_europa**.

Reply:
There is: https://github.com/nasa/ow_europa/pull/16 I guess nobody saw that email because we get so many github notifications.

Reply:
It is better to put a reference in the main PR description to any related PR from another repo. This way no one would (or should) miss that.

Reply:
updated description